src/main/java/org/parabot/core/ui/BotUI.java

```diff
@@ -64,11 +65,34 @@ public static BotUI getInstance() {
         return instance;
     }
 
+    public static boolean deleteDirectory(File directory) {
+        if (directory.exists()) {
+            File[] files = directory.listFiles();
+            if (files != null) {
+                for (int i = 0; i < files.length; i++) {
+                    if (files[i].isDirectory()) {
+                        deleteDirectory(files[i]);
+                    } else {
+                        files[i].delete();
+                    }
+                }
+            }
+        }
+        return (directory.delete());
+    }
+
+    public static void createDirectory(File directory) {
```

Comment:
Call [this](https://github.com/Parabot/Parabot/blob/master/src/main/java/org/parabot/core/Directories.java#L51) method instead